snap/hooks/install

```diff
@@ -10,14 +10,15 @@ SNAP_CURRENT=${SNAP/%$SNAP_REVISION/current}
 # into $SNAP_DATA/config
 # note that app-service-configurable is handled separately
 mkdir -p "$SNAP_DATA/config"
-for service in edgex-mongo security-file-token-provider security-proxy-setup security-secrets-setup security-secretstore-setup core-command core-data core-metadata support-logging support-notifications support-scheduler sys-mgmt-agent device-virtual; do
+for service in edgex-mongo security-file-token-provider security-proxy-setup security-secrets-setup security-secretstore-setup core-command core-data core-metadata support-logging support-notifications support-scheduler sys-mgmt-agent device-virtual security-secretstore-read; do
```

Comment:
is this a new service? I don't see the service defined in the snapcraft.yaml ?

Reply:
Yes, this is a new service in edgex-go which reads the redis password from vault, and writes it to the redis password file. It's included in the edgex-go part's build-override logic (see the first for loop which iterates over the all the services to copy the binaries and configuration files).

Reply:
I guess I was more wondering why it's not a proper app/service in the yaml?

Reply:
Because it's only ever called once from the wrapper script to read the Redis password from Vault, and write it to disk. I treated this the same as security-file-token-provider which is launched from security-secret-store-setup. Ideally, Redis should just grow support for reading secrets directly from Vault, but that wasn't scoped for this release.

Reply:
ok, thanks for the explanation